game/script-topics.rpy

```diff
@@ -1824,6 +1827,79 @@ label talk_sleeping_well:
 
     return
 
+# Natsuki discusses her gaming habits
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_gaming",
+            unlocked=True,
+            prompt="Are you into video games?",
+            category=["Natsuki", "Media"],
+            player_says=True,
+            affinity_range=(jn_affinity.NORMAL, jn_affinity.LOVE),
+            location="classroom"
+        ),
+        topic_group=TOPIC_TYPE_NORMAL
+    )
+
+label talk_gaming:
+    n "Gaming?" 
+    n "Well...{w=0.3} duh!"
+    n "You bet I'm into gaming,{w=0.1} [player]!"
+    n "I wouldn't say I'm the most active player...{w=0.2} but I definitely do my share of button mashing."
+    n "Hmm..."
+    n "I don't think I even need to ask,{w=0.1} but..."
+    menu:
+        n "What about you,{w=0.1} [player]?{w=0.2} Do you play often?"
+
+        "Absolutely!":
+            $ persistent.jn_player_gaming_frequency = "High"
+            n "Yep!{w=0.2} Just as I suspected..."
+            n "[player] is a mega-dork."
+            n "Ahaha!"
+            n "Relax,{w=0.1} [player]!" 
```

Comment:
Line 1861: Could add "I'm just messing with you!" after the existing dialogue on this line.